persistence/msmq/subscription_config_core_[,4].partial.md

```diff
@@ -1,4 +1,4 @@
-WARNING: Storing subscriptions in MSMQ is not suitable when endpoints are scaled-out, because the subscription queue cannot be shared among multiple endpoints.
+WARNING: Storing subscriptions in MSMQ **must not** be used when scaling out across multiple machines or running side-by-side on the same machine. The subscription queue cannot be shared among multiple endpoints instances for the same endpoint.
```

Comment:
I found "is not suitable" better than "must not", but could be because I'm not native English. "must not" doesn't sound right to me.